scripts/SeafoamIslandsB3F.asm

```diff
@@ -50,12 +50,13 @@ Seafoam4HolesCoords:
 	db -1 ; end
 
 SeafoamIslandsB3F_ScriptPointers:
-	dw SeafoamIslands4Script0
-	dw SeafoamIslands4Script1
-	dw SeafoamIslands4Script2
-	dw SeafoamIslands4Script3
+	def_script_pointers
+	dw_const SeafoamIslandsB3FDefaultScript,       SCRIPT_SEAFOAMISLANDSB3F_DEFAULT
+	dw_const SeafoamIslandsB3FObjectMoving1Script, SCRIPT_SEAFOAMISLANDSB3F_OBJECT_MOVING1
+	dw_const SeafoamIslandsB3FMoveObjectScript,    SCRIPT_SEAFOAMISLANDSB3F_MOVE_OBJECT
+	dw_const SeafoamIslandsB3FObjectMoving2Script, SCRIPT_SEAFOAMISLANDSB3F_OBJECT_MOVING2
```

Comment:
These scripts and the rest of the SeafoamIslands scripts probably need better names than this... If someone wants to take a look; let me know.

Reply:
It all has to do with moving the boulders around... I didn't have the brain cells to feel like trying go into detail with those scripts... They were also very, very redundant here. Many routines that do practically the same thing.